BranchSDK/src/BranchIO/PropertyManager.h

```diff
@@ -107,13 +110,48 @@ class BRANCHIO_DLL_EXPORT PropertyManager : protected JSONObject, public virtual
      */
     std::string getStringProperty(const char *name);
 
+    /**
+     * Load a string property from storage.
+     * @param path Path from which this key will be loaded
+     * @param key Key to load
+     * @param defValue Default value if not found
+     * @return String from storage, or default value if not found.
+     */
+    std::string loadString(const char *path, const char* key, const std::string &defValue);
+
+    /**
+     * Save a property to storage.
+     * @param path Path to which this key will be saved
+     * @param key Key name
+     * @param value Key value
+     */
+    void saveString(const char *path, const char *key, const std::string &value);
+
+    /**
+     * Load a string property from storage.
+     * @param path Path from which this key will be loaded
+     * @param key Key to load
+     * @param defValue bool value from storage, or default value if not found.
+     * @return true if the value was found
```

Comment:
I believe the current implementation would not return true if the value that was found was `false`

Reply:
Reworked how storage can use default variables.   It was really not useful to have a `has()` method to check if a key exists, and also return a boolean if the variable was loaded from storage.   I updated the method signature to allow for a defaultValue instead, when the key was not found.

Reply:
💯